core/src/io/anuke/mindustry/core/NetClient.java

```diff
@@ -198,6 +199,16 @@ public static String colorizeName(int id, String name){
         return "[#" + player.color.toString().toUpperCase() + "]" + name;
     }
 
+    @Remote(called = Loc.client, variants = Variant.one)
+    public static void onConnect(String ip, int port){
+        netClient.disconnectQuietly();
+        state.set(State.menu);
+        logic.reset();
+
+        Vars.netClient.beginConnecting();
+        net.connect(ip, port, Actions::hide);
```

Comment:
What is `Actions::hide` for? There's nothing to hide here.

Reply:
Can use ``net.connect(ip, port, null);``?

Reply:
No, but there's no reason to use an empty UI action.